components/automate-dex/habitat/templates/config.yml

```diff
@@ -203,7 +203,9 @@ connectors:
       {{- if cfg.service.external_fqdn}}
       redirectURI: https://{{cfg.service.external_fqdn}}/dex/callback
       {{- else}}
-      redirectURI: http://{{cfg.service.host}}:{{cfg.service.port}}/dex/callback
+      # This is likely not the behavior you want. You should set external
+      # fqdn
```

Comment:
This comment has me nervous because it seems like a message to the user but the user is unlikely to read the comments here :D.

Reply:
i'm about 51% sure nobody will ever make it here